src/ansys/fluent/core/meshing/meshing_workflow.py

```diff
@@ -46,14 +56,45 @@ def __init__(
             Underlying workflow object.
         meshing : PyMenuGeneric
             The meshing object.
+        name: str
+            Workflow name to initialize it.
+        identifier: str
+            Workflow name to identify it from global settings.
+        fluent_version: FluentVersion
+            Version information of the current fluent session.
```

Comment:
```suggestion
            Workflow name for this instance.
        identifier: str
            Workflow name to identify it from global settings.
        fluent_version: FluentVersion
            Version of Fluent to use in this session.
```
I'm a bit confused by the description for the "Identifier" parameter. Maybe "Name of the workflow in the global settings." I'm probably misunderstanding the purpose, but you can see how clarifying the description would be helpful to users.